tools/train_net.py

```diff
@@ -29,6 +29,17 @@
 from segmentation.utils import AverageMeter
 from segmentation.utils.utils import get_loss_info_str
 
+def to_cuda(batch, device):
+    if type(batch) == torch.Tensor:
+        batch = batch.to(device)
+    elif type(batch) == dict:
+        for key in batch.keys():
+            batch[key] = to_cuda(batch[key], device)
+    elif type(batch) == list:
+        for i in range(len(batch)):
+            batch[i] = to_cuda(batch[i], device)
+    return batch
+
```

Comment:
Could you move this function to `utils` in [utils.py](https://github.com/bowenc0221/panoptic-deeplab/blob/master/segmentation/utils/utils.py)?